packages/core/src/Stores/contract-replay-store.js

```diff
@@ -282,7 +282,7 @@ export default class ContractReplayStore extends BaseStore {
                 contractSold(this.root_store.client.currency, response.sell.sold_for, Money)
             );
 
-            RudderStack.track(event_names.reports, {
+            Analytics.trackEvent(event_names.reports, {
```

Comment:
getRudderstackConfig not needed anymore since we have proper types for each event by this type TEvents[T]